m/processing_tools/preprocessGliderData.m

```diff
@@ -739,6 +757,9 @@
     if isfield(ctd_choice_list, 'time')
       time_ctd_field = ctd_choice.time;
     end
+    if isfield(ctd_choice_list, 'oxygen') % -FC
+      oxygen_ctd_field = ctd_choice.oxygen;
+    end    
```

Comment:
Here I don't understand what you mean? Do you mean I should remove `% -FC` ? 

If yes, sorry I usually work alone with my scripts and this help me to quickly screen what I am doing. I will remove.